spec.md

```diff
@@ -462,39 +504,21 @@ schema being used.
           "create": {
             "parameters": {
               "$schema": "http://json-schema.org/draft-04/schema#",
-              "type": "object",
-              "properties": {
-                "billing-account": {
-                  "description": "Billing account number used to charge use of shared fake server.",
-                  "type": "string"
-                }
-              }
+              "$ref": "http://www.openservicebrokerapi.org/v2/catalog/root/#BillingAccountParameters"
```

Comment:
I think this URI is just an example, right? If so, i'd prefer to not use the OSBAPI URI, otherwise people might think its an ok thing to do, rather than use one specific to their service

Reply:
It kind of is ok. It does no harm, we say you can't have external URI loads.

Reply:
its not a question of harm as much as its about ownership. Technically, we own our URI, examples that suggest other people should use our URI for their own data is bad precedent.

Reply:
the OSB spec says $schema is required :(

Reply:
Right -- in the face of $ref usage patterns we'll need to re-think this a bit.
If the $ref referenced a root schema document with a $schema I think we would be fine. Unfortunately I think the more common usage pattern will be to reference a part of a schema document via pointer or $id.   
One possibility is to infer the $schema from the referenced document if expecting a root schema. Another alternative is to use $ref one-level down so we don't overwrite the root schema documents $schema.

Reply:
Thinking about this more If $ref is using JSON Schema "delegation" everything works out just fine meaning...
```
"parameters": {
    "$ref": "http://www.openservicebrokerapi.org/v2/catalog/root/#BillingAccountParameters"
 }
```
 ... is acceptable.